packages/kit/test/apps/options/svelte.config.js

```diff
@@ -17,7 +17,16 @@ const config = {
 			build: {
 				minify: false
 			},
-			clearScreen: false
+			clearScreen: false,
+			server: {
+				hmr: {
```

Comment:
I don't think I've seen these options set in any of the other tests. worth a comment explaining?

Reply:
left over from #2145, i just assumed it was there for a reason. we can probably remove